subsys/random/rand32_timer.c

```diff
@@ -62,20 +62,16 @@ uint32_t z_impl_sys_rand32_get(void)
 
 void z_impl_sys_rand_get(void *dst, size_t outlen)
 {
-	uint32_t len = 0;
-	uint32_t blocksize = 4;
+	uint8_t *udst = (uint8_t *)dst;
```

Comment:
Typecast not needed to assign a `void *` pointer. Is this required by a guideline?